barretenberg/cpp/CMakePresets.json

```diff
@@ -351,8 +351,7 @@
         "acvm_backend.wasm",
         "barretenberg",
         "wasi",
-        "env",
-        "honk_tests"
```

Comment:
we don't have honk_tests anymore i think and the `wasm` CI check was failing because of this